### ISSUE TITLE
Domain certificate expired Sept 29, 2022

### ISSUE THREAD

**Issue by royhandy** [Flamenco/stackeditpro#19]
![image](https://user-images.githubusercontent.com/25946088/203223468-acff003b-6634-4be4-84c6-902171f2726b.png)

**Comment by Flamenco** [comment]
@royhandy  I think this might be a Chrome bug displaying old certificate.  I open in Firefox and using online SSL checker, and the date is different.  My Chrome certificate details shows valid, but when viewing details displays the old date as well.

Can you please try using a different browser?

**Comment by Flamenco** [comment]
<img width="1027" alt="Screenshot 2022-11-22 at 3 06 20 PM" src="https://user-images.githubusercontent.com/819940/203411001-5ab60b84-3c6a-4dc9-a896-34f72c793d36.png">

**Comment by royhandy** [comment]
> @royhandy I think this might be a Chrome bug displaying old certificate. I open in Firefox and using online SSL checker, and the date is different. My Chrome certificate details shows valid, but when viewing details displays the old date as well.
> 
> Can you please try using a different browser?

I received a warning using Safari.   I just checked using Edge (Windows) on a profile that I actively use stackeditpro on and it shows secure but when I click to view the certificate is shows the same expiry (Sept 2022).  I switched to another profile in Edge and it shows Sept 2023.   I'm guessing a caching issue somewhere - anyway, glad to know it's current.  I'll just ignore the warning for now

![image](https://user-images.githubusercontent.com/25946088/203415875-179e7a1b-22af-482f-b417-49231653464f.png)
